blockchain/blockchain.go

```diff
@@ -62,7 +62,9 @@ func checkBlockVersion(protocolVersion string) error {
 		return err
 	}
 
-	if blockVer.GreaterThan(supportedStarknetVersion) {
+	// We ignore changes in patch part of the version
+	blockVerMM, suportedVerMM := blockVer.IncMinor(), SupportedStarknetVersion.IncMinor()
+	if blockVerMM.GreaterThan(&suportedVerMM) {
```

Comment:
wdyt about adding this function:
```
func CopyWithoutPatch(v *semver.Version) *semver.Version {
	if v == nil {
		return nil
	}

	return semver.New(v.Major(), v.Minor(), 0, v.Prerelease(), v.Metadata())
}
```
to `utils` ?

Reply:
we can update function name later, but I think it would be clear how we treat block version and supported versions